apps/storybook/src/gp2-components/UserCardInfo.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { gp2 as gp2Fixtures } from '@asap-hub/fixtures';
+
+import UserCardInfo from '@asap-hub/gp2-components/src/molecules/UserCardInfo';
+
+import UserRegion from '@asap-hub/gp2-components/src/molecules/UserRegion';
+
+import { ComponentProps } from 'react';
+
+const { createUserResponse } = gp2Fixtures;
+
+export default {
+  title: 'GP2 / Molecules / User Card Info',
+  component: UserCardInfo,
+};
+
+type UserCardInfoProps = Pick<ComponentProps<typeof UserRegion>, 'region'> & {
+  workingGroups: { id: string; name: string }[];
+  projects: { id: string; name: string }[];
+  role: string;
+};
```

Comment:
Looks wrong to be defining the props here. 

maybe we need something like:

```
ComponentProps<typeof UserCardInfo>
```

Reply:
we shouldn't really need to type cast it tho

Reply:
is there any particular prop that it complains about?